### ISSUE TITLE
focal loss test nan

### ISSUE THREAD

**Issue by JunMa11** [Hsuxu/Loss_ToolBox-PyTorch#7]
Dear @Hsuxu ,
Thanks for sharing the code.
When running  `FocalLoss_test.py`,  all the outputs are `nan`.
https://github.com/Hsuxu/Loss_ToolBox-PyTorch/blob/2f2b0fe6b65398c16cc31a9fb1b6447af9e62d4e/FocalLoss/FocalLoss_test.py#L36

![image](https://user-images.githubusercontent.com/19947331/62064066-142dd880-b1fa-11e9-9179-947b770ff8ae.png)

Is there anything wrong?

**Comment by Hsuxu** [comment]
Due to issue #3 I have remove `softmax` function, you may add `output=F.softmax(output,dim=1)` after   
https://github.com/Hsuxu/Loss_ToolBox-PyTorch/blob/7c1f9d86e7d28670fd708d0a32e89bd8ba42ee50/FocalLoss/FocalLoss_test.py#L34

**Comment by JunMa11** [comment]
Got it! Thanks for your reply very much.
